src/main/java/dev/sefiraat/sefilib/world/TimePeriod.java

```diff
@@ -141,7 +163,7 @@ public static boolean isDay(@Nonnull World world) {
      * @param time The time to check against
      * @return true if the given time would be day in the overworld.
      */
-    public static boolean isDay(Long time) {
+    public static boolean isDay(long time) {
```

Comment:
While I don’t imagine this has been used by many, if anyone, this is still and API and this will change the method signature so each of the old ones will need to stay for a time marked with `@depricated` and the ‘new’ methods alongside. Sorry it’s awkward :)